components/Encode/Encode.tsx

```diff
@@ -32,6 +32,14 @@ const Encode: React.FC<Props> = ({ web3 }) => {
                 onChange={() => setMode(TRANSACTION_TYPES.SET_DATA)}
               />
               setData
+              <a
+                href="https://github.com/ERC725Alliance/ERC725/blob/main/docs/ERC-725.md#setdata"
```

Comment:
I'd recommend this page instead

https://docs.lukso.tech/contracts/contracts/ERC725/#setdata

Wdyt?